neofetch

```diff
@@ -10792,6 +10792,34 @@ dKK${c1}KKKKKKKKKK;.;oOKx,..${c2}^${c1}..;kKKK0.${c2}  dKd
      'l0Kk:.              .;xK0l'
         'lkK0xl:;,,,,;:ldO0kl'
             '^:ldxkkkkxdl:^'
+EOF
+        ;;
+        "parch"* | "Parch"* | "PARCH"*)
+            set_colors 4 6
+            read -rd '' ascii_data <<'EOF'                                                                     
+                                                                                
+                                                                                
+‌		  ‌+s;;:o-
+              s/sssso+/-'+\
```

Comment:
You need to escape all backslashes (replace them with double backslashes) for it to read properly.

```suggestion
              s/sssso+/-'+\\
```

Before escaping backslash:

![image](https://github.com/dylanaraps/neofetch/assets/22280294/80af0b25-61a8-4706-a09a-2541fb8603e7)

After escaping backslash:

![image](https://github.com/dylanaraps/neofetch/assets/22280294/1cbcf1da-a367-4514-a32d-3f70a10fb843)